velox/exec/tests/utils/HiveConnectorTestBase.h

```diff
@@ -43,23 +41,32 @@ class HiveConnectorTestBase : public OperatorTestBase {
   void resetHiveConnector(
       const std::shared_ptr<const config::ConfigBase>& config);
 
-  void writeToFile(const std::string& filePath, RowVectorPtr vector);
+  void writeToFile(
+      const std::string& filePath,
+      const RowVectorPtr vector,
+      const dwio::common::FileFormat& fileFormat =
```

Comment:
For the fileFormat the parameter doesn't need to be a const & as it is an enum (so it similar to numeric types).

Reply:
Changing to `dwio::common::FileFormat fileFormat`